multilinear_extensions/src/expression.rs

```diff
@@ -970,7 +975,7 @@ pub fn wit_infer_by_expr<'a, E: ExtensionField>(
     expr: &Expression<E>,
 ) -> ArcMultilinearExtension<'a, E> {
     expr.evaluate_with_instance::<ArcMultilinearExtension<'a, E>>(
-        &|f| fixed[f.0].clone(),
```

Comment:
is this change only for debug purpose?

Reply:
I found currently there are no `Fixed` vectors in used, since all `Fixed` polys will be converted to `WitIn` by the constraint systems (both in ceno_zkvm and gkr_iop). It seems not well-defined to put `fixed[]` here.

Reply:
I just restored this line in https://github.com/scroll-tech/ceno/pull/970/commits/5682bd995729ad270e9a4995fbf3f6880ad9ec0b. I previously changed it because I found currently there are no `Fixed` vectors in used, since all `Fixed` polys will be converted to `WitIn` by the constraint systems (both in ceno_zkvm and gkr_iop). So it used to confuse me what to put when using it in `gkr_iop` lol.